src/main/java/com/minecolonies/coremod/entity/ai/mobs/util/MobEventsUtils.java

```diff
@@ -0,0 +1,231 @@
+package com.minecolonies.coremod.entity.ai.mobs.util;
+
+import com.minecolonies.api.configuration.Configurations;
+import com.minecolonies.api.util.LanguageHandler;
+import com.minecolonies.coremod.colony.CitizenData;
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.ColonyView;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utils for Colony mob events
+ */
+public class MobEventsUtils
+{
+    private static final String BARBARIAN = "minecolonies.Barbarian";
+    private static final String ARCHER    = "minecolonies.ArcherBarbarian";
+    private static final String CHIEF     = "minecolonies.ChiefBarbarian";
+
+    private static final float  WHOLE_CIRCLE                 = 360.0F;
+    private static final float  HALF_A_CIRCLE                = 180F;
+    private static final int    MAX_SIZE                     = Configurations.maxBarbarianHordeSize;
+    private static final double BARBARIANS_MULTIPLIER        = 0.5;
+    private static final double ARCHER_BARBARIANS_MULTIPLIER = 0.25;
+    private static final double CHIEF_BARBARIANS_MULTIPLIER  = 0.1;
+    private static final int    PREFERRED_MAX_HORDE_SIZE     = 40;
+    private static final int    PREFERRED_MAX_BARBARIANS     = 22;
+    private static final int    PREFERRED_MAX_ARCHERS        = 16;
+    private static final int    PREFERRED_MAX_CHIEFS         = 2;
+    private static final int    MIN_CITIZENS_FOR_RAID        = 5;
+    private static final int    HALF_MINECRAFT_DAY           = 12000;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add underscores to this numeric value for readability [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2148)